terraform.tfstate

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
why is that committed? I've seen that in .gitignore ....

Reply:
By a mistake, thx for catching this!
File has been already removed.